hubgrep/lib/fetch_results.py

```diff
@@ -50,7 +52,7 @@ def _normalize(results):
 
 def fetch_concurrently(
     keywords, hosting_service_interfaces: List[HostingServiceInterface]
-):
+) -> (List[SearchResult], (HostingServiceInterface, CachedResponse)):
```

Comment:
as explained above: we still return a list of results and a list of errors, but the errors are a tuple of the hosting_interface and the corresponding response